taxonomies/additives.txt

```diff
@@ -18451,7 +18451,7 @@ cs:E951, Aspartam, E 951, NutraSweet
 da:E951, Aspartam
 de:E951, Aspartam, E 951, Canderel, Asphartam, NutraSweet
 el:E951, Ασπαρταμη
-es:E951, Aspartamo, E 951, Espartame, Aspartame
+es:E951, Aspartamo, E 951, Espartame, Aspartame, fenilalanina
```

Comment:
This we should not add, fenilanina is contained in aspartam, but it's not the same thing.

Instead we should add a translation in labels.txt for "contains a source of phenylalanin" in Spanish.

That's what we have today in labels.txt:

en:Contains a source of phenylalanine
ca:Conté una font de fenilalanina
de:Enthält eine Phenylalaninquelle, Das produkt enthält eine Phenylalaninquelle
fi:Sisältää fenyylialaniinin lähteen
fr:Contient une source de phénylalanine
it:Contiene una fonte di fenilalanina

Reply:
ok